TemotoMetadata TemotoMetadata
string loadTarget
---
#result
TemotoMetadata TemotoMetadata
string loadMessage